### ISSUE TITLE
Hamburger menu has no label

### ISSUE THREAD

**Issue by bxie** [codeandcognition/koconut#428]
Hamburger menu in top right is missing label, so it is no usable w/ screen reader.

Should add label.

![image](https://user-images.githubusercontent.com/3218912/87485309-57d94800-c5ed-11ea-90cd-99e99160864a.png)
